libs/events/src/EventEmitter.ts

```diff
@@ -1,34 +1,47 @@
-import { EventNames, EventPayloads } from './types'
+import { CowEvents, EventPayloads as CowEventPayloads } from './types'
```

Comment:
Did you mean to rename the source `EventPayloads` type as well?

Reply:
Yes, i saw this later, is not present in the most recent PRs